python/HeaderService/hslib_salobj.py

```diff
@@ -531,6 +547,11 @@ def extract_from_myData(self, keyword, myData, sep=":"):
             key = self.config.telemetry[keyword]['array_keyname']
             # Extract only the requested key from the dictionary
             extracted_payload = dict(zip(keywords, split_esc(payload, sep)))[key]
+        # Case 3 -- enumeration using idl libraries
+        elif self.config.telemetry[keyword]['array'] == 'enum':
+            Device = self.config.telemetry[keyword]['device']
```

Comment:
Capital case is reserved for classes. Use lower case for this.

Reply:
Fixed.